### PR TITLE
fixes, rename miopen/miopengemm, typos, 2.6 ebuilds

### PR DESCRIPTION
**Pull request by devang**
Kind of long, sorry.

### CODE REVIEW

File: sci-libs/rocFFT/rocFFT-2.6.0.ebuild [hunk 1]
```diff
@@ -18,58 +18,36 @@ IUSE="+gfx803 gfx900 gfx906"
...
-
-	eapply "${FILESDIR}/rocFFT-library_src_CMakeListstxt.patch"
-	eapply "${FILESDIR}/rocFFT-library_src_device_CMakeListstxt.patch"
-
-	sed -e "s:rocm_install_symlink_subdir( rocfft ):#rocm_install_symlink_subdir( rocfft ):" -i ${S}/library/src/CMakeLists.txt
-	sed -e "s: PREFIX rocfft:# PREFIX rocfft:" -i ${S}/library/src/CMakeLists.txt
-
-	sed -e "s: PREFIX rocfft:# PREFIX rocfft:" -i ${S}/library/src/device/CMakeLists.txt
-	sed -e "s:rocm_install_symlink_subdir( rocfft ):#rocm_install_symlink_subdir( rocfft ):" -i ${S}/library/src/device/CMakeLists.txt
-
```
[the remark this answers is not in the document]
Comment (devang) [reply sci-libs/rocFFT/rocFFT-2.6.0.ebuild]: I can add these two sed lines back if that's all it takes for FHS. The build was failing for me, so I redid everything using cmake-utils.

[the remark this answers is not in the document]
Comment (justxi) [reply sci-libs/rocFFT/rocFFT-2.6.0.ebuild]: If it fails to build, please comment it out. Then we can check it later.

File: dev-libs/rccl/rccl-2.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	if use gfx803; then
+		CurrentISA="803"
+	fi
+	if use gfx900; then
+		CurrentISA="900"
+	fi
+	if use gfx906; then
+		CurrentISA="906"
+	fi
+	cmake-utils_src_prepare
```
[the remark this answers is not in the document]
Comment (devang) [reply dev-libs/rccl/rccl-2.6.0.ebuild]: thanks, done

File: sci-libs/rocFFT/rocFFT-2.6.0.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (devang) [reply sci-libs/rocFFT/rocFFT-2.6.0.ebuild]: ok, I can put back the lines as comments.

**Comment by justxi** [comment]
> Kind of long, sorry.

Never mind. Thanks for the work.

**Comment by devang** [comment]
np, let me know if there is anything else.

**Pull request merged**